batch/snippets/src/main/java/com/example/batch/CreateBatchRunnableLable.java

```diff
@@ -0,0 +1,138 @@
+  public static Job createBatchRunnableLable(String projectId, String region, String jobName,
+                   String labelName1, String labelValue1, String labelName2, String labelValue2)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (BatchServiceClient batchServiceClient = BatchServiceClient.create()) {
+
+      // Define what will be done as part of the job.
+      Runnable runnable =
+          Runnable.newBuilder()
+              .setContainer(
+                  Runnable.Container.newBuilder()
+                      .setImageUri("gcr.io/google-containers/busybox")
+                      .setEntrypoint("/bin/sh")
+                      .addCommands("-c")
+                      .addCommands(
+                          "echo Hello world! This is task ${BATCH_TASK_INDEX}. "
+                              + "This job has a total of ${BATCH_TASK_COUNT} tasks.")
+                      .build())
+              // Label and its value to be applied to the first runnable.
+              .putLabels(labelName1, labelValue1)
+              .setScript(Runnable.Script.newBuilder()
+              .setText("echo Hello world! This is task ${BATCH_TASK_INDEX}. ").build())
+              // Label and its value to be applied to the second runnable.
+              .putLabels(labelName2, labelValue2)
+              .build();
+
+      // We can specify what resources are requested by each task.
+      ComputeResource computeResource =
+          ComputeResource.newBuilder()
+              // In milliseconds per cpu-second. This means the task requires 50% of a single CPUs.
+              .setCpuMilli(2000)
+              // In MiB.
+              .setMemoryMib(2000)
+              .build();
+
+      TaskSpec task =
+          TaskSpec.newBuilder()
+              // Jobs can be divided into tasks. In this case, we have only one task.
+              .addRunnables(runnable)
+              .setComputeResource(computeResource)
+              .setMaxRetryCount(2)
+              .setMaxRunDuration(Duration.newBuilder().setSeconds(3600).build())
+              .build();
+
+      // Tasks are grouped inside a job using TaskGroups.
+      // Currently, it's possible to have only one task group.
+      TaskGroup taskGroup = TaskGroup.newBuilder().setTaskCount(1).setTaskSpec(task).build();
+
+      Job job =
+          Job.newBuilder()
+              .addTaskGroups(taskGroup)
+              // We use Cloud Logging as it's an out of the box available option.
+              .setLogsPolicy(LogsPolicy.newBuilder()
+              .setDestination(LogsPolicy.Destination.CLOUD_LOGGING).build())
+              .build();
+
+      CreateJobRequest createJobRequest =
+          CreateJobRequest.newBuilder()
+              // The job's parent is the region in which the job will run.
```

Comment:
Parent doesn't refer to only the region. Kindly update the comment.

Reply:
Maybe this is implicit. I'm thinking if we need this comment in the first place.